### PR TITLE
Updated SWESARR

### PR DESCRIPTION
**Pull request by db1950**
Trying again with a new branch! Really gotta up my git game one of these days ❌

### CODE REVIEW

File: book/tutorials/sar/swesarr.ipynb [hunk 1]
```diff
@@ -468,7 +492,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!wget --quiet https://n5eil01u.ecs.nsidc.org/SNOWEX/SNEX20_SWESARR_TB.001/2020.02.11/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKKa225H_v01.csv -O {output_dir}/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKuKa225H_v03.csv"
+    "!wget --user=this --password=that --quiet https://n5eil01u.ecs.nsidc.org/SNOWEX/SNEX20_SWESARR_TB.001/2020.02.11/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKKa225H_v01.csv -O {output_dir}/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKuKa225H_v03.csv"
```
Comment (scottyhq) [inline book/tutorials/sar/swesarr.ipynb:495]: I'd prefer not to change this. We instructed everyone to set up a ~/.netrc file (), and out CI system that executes and renders the website relies on such a file being there. Adding the user and password flags means this command will fail so the notebook won't render. https://snowex-hackweek.github.io/website/preliminary/earthdata.html#configure-programmatic-access-to-nasa-servers

**Review by scottyhq — CHANGES_REQUESTED** [review]
can merge early tmrw morning after the requested change

File: book/tutorials/sar/swesarr.ipynb [hunk 2]
```diff
@@ -69,12 +68,32 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5cb68a4f-3e5b-4d29-882f-b02cdc524392",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Audio,Image, YouTubeVideo;    id='5hVQusosGSg';    YouTubeVideo(id=id,width=600,height=300,start=210,end=238)\n",
```
Comment (scottyhq) [inline book/tutorials/sar/swesarr.ipynb:81]: nice! didn't know you could do this. i'm not 100% sure the video will show up on the website, but definitely works when running the notebook interactively.

File: book/tutorials/sar/swesarr.ipynb (hunk 1 above)
Comment (db1950) [reply book/tutorials/sar/swesarr.ipynb:495]: Hi Scott! I absolutely did not mean to change that. I just noticed I accidentally left it in while doing a practice run an hour ago. Give me just a second and I'll make a new pull request

File: book/tutorials/sar/swesarr.ipynb (hunk 2 above)
Comment (db1950) [reply book/tutorials/sar/swesarr.ipynb:81]: I thought it was really neat! I would prefer to just code it in HTML, but I couldn't quite get the **stop** flag to work for me.

**Review by scottyhq — APPROVED** [review]

**Pull request merged**